### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by Punit-Choudhary** [openAOD/Python-PatternHouse#848] [Q&A] [answered]
Hey admins 👋 

Do you have any plan of marking contributions for `Hacktoberfest` PR

Thanks

**Comment by aryashah2k** [comment]
Hey buddy @Punit-Choudhary , I dont know why did this message not appear on my notifications ! Do tag me next time!

Yes I would love to take this project and open it to hacktoberfest participants, can you help me out if you know the procedure to apply so?

**Comment by Punit-Choudhary** [comment]
Add hacktoberfest label to the issue, don't spam.. otherwise it won't get counted.. and you can't solve that issue on your own.

**Reply by aryashah2k** [reply]
Great, and do I need to formally register this organization with the Hacktoberfest event?

**Comment by aryashah2k** [comment, marked as answer]
Great News, Our Organization is participating in Hacktoberfest and I have registered myself as the Maintainer. Looking forward to see your contributions throughout the month of October and beyond. Cheers @Punit-Choudhary ! Do spread the org among your friends so they can benefit from this season of open source too! 

I will be assigning the hacktoberfest-accepted label to all the successfull PRs myself.
